src/components/CreateSecret/Form/Annotations.js

```diff
@@ -40,6 +43,13 @@ const Annotations = props => {
     defaultMessage: 'Docker Registry'
   });
 
+  function determineServerURLValue(annotation) {
+    console.log(annotation);
+    if (annotation.access === 'git') {
+      return defaultGithubServerURL;
+    }
+    return defaultDockerServerURL;
+  }
```

Comment:
with this change I can no longer edit the URL
we should only use the default value if `annotation.value` is not set / a default value (i.e. switching from git to docker)

Reply:
Thanks for taking a look, updated - have a look at the gotcha I've mentioned though please 👀